### PR TITLE
Fix invalid PHP code samples in utests

### PR DESCRIPTION
**Pull request by SpacePossum**
@see https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1538

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
:+1:

File: Symfony/CS/Tests/Fixer/Symfony/NewWithBracesFixerTest.php [hunk 1]
```diff
@@ -38,15 +38,42 @@ public function test54($expected, $input = null)
...
+            ),
+            array(
+                '<?php $self = new self();',
+                '<?php $self = new self;',
+            ),
+            array(
+                '<?php $static = new static();',
+                '<?php $static = new static;',
+            ),
+            //array( was this valid in any PHP version? https://bugs.php.net/bug.php?id=36221
```
Comment (keradus) [inline Symfony/CS/Tests/Fixer/Symfony/NewWithBracesFixerTest.php]: Nope, this is OK:
```php
<?php

class Foo
{
    public static function create()
    {
        $x = __CLASS__;
        return new $x();
    }
}

echo get_class(Foo::create());
```

but this is not:
<?php

class Foo
{
    public static function create()
    {
        return new __CLASS__();
    }
}

echo get_class(Foo::create());

Comment (keradus) [inline Symfony/CS/Tests/Fixer/Symfony/NewWithBracesFixerTest.php]: please remove that case and PR is good to go

Comment (SpacePossum) [inline Symfony/CS/Tests/Fixer/Symfony/NewWithBracesFixerTest.php]: :+1: 
done, do you want me to squash this one?

Comment (keradus) [inline Symfony/CS/Tests/Fixer/Symfony/NewWithBracesFixerTest.php]: no need to

**Comment by gharlan** [comment]
:+1:

**Comment by keradus** [comment]
Thank you @SpacePossum.

**Pull request closed without merging**